src/main/java/wolox/training/config/PersistenceConfig.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.config;
+
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EntityScan("wolox.training.models")
+public class PersistenceConfig {
```

Comment:
Why do you need this class? I think that it does nothing.

Reply:
This class was added by recommendation, but if you want remove it, i do.

Reply:
They also recommend using the notation for repositories and it can be used in this class.

Reply:
I like to follow the YAGNI principle: https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

If this class serves no purpose, it's best to not add it until it is necessary, to reduce code complexity.